### PR TITLE
[MIG] base_comment_template: Migration to 12.0

### PR DESCRIPTION
**Pull request by rven**

### CODE REVIEW

**Review by bjeficent — APPROVED** [review]
Code review

File: base_comment_template/views/comment_view.xml [hunk 1]
```diff
@@ -2,49 +2,49 @@
...
-        <tree string="Comment Templates">
-          <field name="name"/>
-          <field name="position"/>
-        </tree>
-      </field>
+        <field name="name">account.comment.template.list</field>
+        <field name="model">base.comment.template</field>
+        <field name="priority" eval="6"/>
+        <field name="arch" type="xml">
+            <tree>
```
Comment (bjeficent) [inline base_comment_template/views/comment_view.xml:34]: Why are you removing the string here?

Comment (rven) [reply base_comment_template/views/comment_view.xml:34]: @bjeficent if you run pylint it gives the following output:
Deprecated xml attribute "string" (xml-deprecated-tree-attribute)

https://github.com/OCA/pylint-odoo/issues/146

**Review by AaronHForgeFlow — APPROVED** [review]
Code review

File: base_comment_template/views/comment_view.xml (hunk 1 above)
Comment (bjeficent) [reply base_comment_template/views/comment_view.xml:34]: Oh I didn't know that. Thanks for the quick answer

**Pull request merged**